src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/tests/test_deployment_map.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# pylint: skip-file
+
+import os
+import boto3
+
+from errors import InvalidDeploymentMapError
+from pytest import fixture, raises
+from mock import Mock
+from ..pipeline import Pipeline
+from ..deployment_map import DeploymentMap
+
+
+@fixture
+def cls():
+    return DeploymentMap(
+        parameter_store=None,
+        s3=None,
+        pipeline_name_prefix="adf",
+        map_path="{0}/stubs/stub_deployment_map.yml".format(
+            os.path.dirname(os.path.realpath(__file__))
+        ),
+    )
+
+
+def test_update_deployment_parameters(cls):
+    cls.s3 = Mock()
+    cls.s3.put_object.return_value = None
+
+    pipeline = Pipeline(
+        {
+            "name": "pipeline",
+            "params": {"key": "value"},
+            "targets": [],
+            "default_providers": {
+                "source": {
+                    "name": "codecommit",
+                    "properties": {
+                        "account_id": 111111111111,
+                    },
+                }
+            },
+        }
+    )
+
+    # Targets : [[account_id, account_id], [account_id, account_id]]
+    pipeline.template_dictionary = {
+        "targets": [
+            [
+                [
+                    {
+                        "id": "111111111111",
+                        "name": "some_account",
+                        "path": "/fake/path",
+                        "properties": {},
+                        "provider": {},
+                        "regions": ["eu-west-1"],
+                        "step_name": "",
+                    },
+                ]
+            ]
+        ]
+    }
+
+    cls.update_deployment_parameters(pipeline)
+    assert cls.account_ou_names["some_account"] == "/fake/path"
```

Comment:
Could you add the tests I added in the #484 Fix: Wave deployments PR. That way I can close that one in favour of this one :smile:

See https://github.com/awslabs/aws-deployment-framework/pull/484/commits/18a0492b46ade867baa8f1e7256d72bbbf92f93b